### PR TITLE
Blog-app: Views

### PR DESCRIPTION
**Pull request by tajulafreen**
**In this PR I made the following changes**
- Added :
    - Basic UI created for pages to show all users fetched from the database
    - Basic UI created for pages to show all Posts fetched from the database

### CODE REVIEW

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -1 +1,17 @@
...
+            <h4 class="pt-2">'<%= @post.title %>' by <%= @user.name %></h4>
+            <p class="mt-2"><%= @post.text %></p>
+            <p class="d-flex justify-content-end">Comments <%= @post.comments_counter %>, Likes <%= @post.likes_counter%></p>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-lg-12 border border-dark">
+            <% @comments.each do |comment| %>
+                <p class=""><%= comment.author.name %>: <%= comment.body %> </p>
+            <% end %>
```
Comment (ShadyShawkat) [inline app/views/posts/show.html.erb]: - [ ] Kindly fix the following error that happens whenever the user visits the posts#show page.

image

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,20 @@
...
+                <h4 class="pt-2"><%= link_to post.title, user_post_path(@user, post), class: "text-dark" %></h4>
+                <p class="mt-2"><%= truncate(post.text, length: 200, omission: '...') %></p>
+                <p class="d-flex justify-content-end">Comments <%= post.comments_counter %>, Likes <%= post.likes_counter%></p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-lg-12 border border-dark">
+                <% post.recent_comments.each do |comment| %>
+                    <p class=""><%= comment.author.name %>: <%= comment.body %> </p>
+                <% end %>
```
Comment (ShadyShawkat) [inline app/views/posts/index.html.erb]: - [ ] Kindly fix the following error that happens whenever the user visits the posts#index page.

image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @tajulafreen, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @tajulafreen    , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  . 
Congratulations! 🎉  Keep up the Awesome work 🥂  Keep Rocking 🥇 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯 
- [x] Professional README file 
- [x] App running as intended. 💯 🥇 
- [x] Blog App views implemented correctly




 

 










 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]

**Pull request merged**